hypervisor/arch/x86/guest/pm.c

```diff
@@ -10,6 +10,7 @@
 #include <io.h>
 #include <logmsg.h>
 #include <platform_acpi_info.h>
+#include <guest_pm.h>
```

Comment:
What happens without the header file?

Reply:
there is an violation "No prototype for non-static function. : register_rt_vm_pm1a_ctl_handler"